docs/ql-documentation/learn-ql/cpp/introduce-libraries-cpp.rst

```diff
@@ -4,114 +4,516 @@ Introducing the C/C++ libraries
+| ``T& T::`` **operator=** ``(const T`` **&** ``);``                                                                                                                                                                                                                                                                                                                                                                                          | `CopyAssignmentOperator <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/Function.qll/type.Function$CopyAssignmentOperator.html>`__                                 |                                                                                                                                                                                                                                                                                   |
++---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| ``T& T::`` **operator=** ``(const T`` **&&** ``);``                                                                                                                                                                                                                                                                                                                                                                                         | `MoveAssignmentOperator <https://help.semmle.com/qldoc/cpp/semmle/code/cpp/Function.qll/type.Function$MoveAssignmentOperator.html>`__                                 |                                                                                                                                                                                                                                                                                   |
```

Comment:
Please drop the `const` here. See https://github.com/Semmle/ql/pull/1716#discussion_r312788284.

Reply:
Good catch, thanks.